docs/advanced/kotlin-codegen.md

```diff
@@ -0,0 +1,85 @@
+# About
+
+In addition to the Java code-gen, we also have an experimental Kotlin API that generates more idiomatic Kotlin classes. Instead of generating Java code that can be used in Kotlin, it generates native Kotlin code that leans into techniques that only Kotlin supports. This is split primarily into two parts: the data classes used to serialize responses (both in the server and in the client), and the client query projections
```

Comment:
It's called "experimental" here. What is your plan to get it out of an experimental phase?

Reply:
That's a great question - it's perfectly stable & we've been using it for some time now with no issues. If you have no objections then I'd be happy dropping the label 🤷‍♂️